lesson2/homework.js

```diff
@@ -0,0 +1,30 @@
+
+
+const addNewTask = function(id, title, description, created, updated, dueDate, statusID, userID) {
+  
+};
+
+const changeTaskTitle = function(taskID, newTitle) {
+  
+};
+
+const changeTaskDueDate = function(taskID, newDueDate) {
+  
+};
+
+const changeTaskStatus = function(taskID, newStatus) {
+  
+};
+
+const markTaskAsCompleted = function(taskID, newStatus) {
+  
+};
+
+const deleteTask = function(taskID, taskID) {
```

Comment:
One taskID argument is probably enough :smile:

Reply:
not sure... xD